apitest/src/main/resources/injicertify/SunBirdCNegative/GenerateTokenSunBirdCNeg/GenerateTokenSunBirdCNeg.yml

```diff
@@ -0,0 +1,20 @@
+GenerateTokenSunBirdC:
```

Comment:
this has to be "GenerateTokenSunBirdCNeg"

Reply:
Updated